pkg/mpc-examples/contracts/03-ebu-rebalancer/README.md

```diff
@@ -0,0 +1,35 @@
+# EBU Rebalancer Controller
+
+## Summary
+EBURebalancer is a Managed Pool Controller that has the ability to pause swaps as well as promote rebalancing through the gradual lowering of swap fees. 
+
+## Details
+The EBURebalancer is an extension of the NullController with added ability to rebalance the Managed Pool by enabling swaps, and updating swap fees gradually to a nominal value over the course of 7 days. This lowering of swap fees incentivises arbitrageurs to bring the pool back to its desired weight of 33/33/33.  
```

Comment:
```suggestion
The EbuRebalancer is a Managed Pool Controller that makes the pool a nominally static basket of tokens. It enables anyone to rebalance the Managed Pool by enabling swaps with an extremely high swap fee, ramping the fees down gradually to a nominal value over the course of 7 days. This lowering of swap fees incentivizes arbitrageurs to bring the pool back to its desired weight. 
```

Reply:
Why are we removing detail that the weights will be set to 33/33/33, since the user is not able to set custom weights